spec/controllers/equipment_models_controller_spec.rb

```diff
@@ -90,7 +90,7 @@
     context 'with admin user' do
       before { mock_user_sign_in(UserMock.new(:admin)) }
       context 'successful save' do
-        let!(:model) { FactoryGirl.build_stubbed(:equipment_model) }
+        let!(:model) { FactoryGirl.build(:equipment_model) }
```

Comment:
that can be fixed with an `allow(model).to receive(:update_attribute)` before the request.